solutions/kubernetes/ceph-binary-build.sh

```diff
@@ -46,14 +46,18 @@ function check_params() {
     if [ -z "$CEPH_BRANCH" ]; then echo "CEPH_BRANCH not provided. Using default: quincy";CEPH_BRANCH="quincy"; fi
     if [ -z "$BUILD_OS" ]; then echo "BUILD_OS not provided. Using default: centos";BUILD_OS="centos"; fi
     if [ -z "$BUILD_LOCATION" ]; then echo "BUILD_LOCATION for container to mount not provided. Using default: /var/log/ceph-build";BUILD_LOCATION="/var/log/ceph-build"; fi
+    if [ -z "$REPO_COMPONENT" ]; then echo "REPO_COMPONENT for ceph repo not provided. Using default: ceph";REPO_COMPONENT="ceph"; fi
     if [ -z "$MOUNT" ]; then echo "MOUNT for uploading packages is not provided. Using default: cortx-storage.colo.seagate.com:/mnt/data1/releases/ceph";MOUNT="cortx-storage.colo.seagate.com:/mnt/data1/releases/ceph"; fi
```

Comment:
should we give it as a input parameter to user? can't we keep it as a constant variable?

CC- @shailesh-vaidya

Reply:
With same job user can build both ceph and cortx-rgw repo. This solution enables to store the generated packages on cortx-storage in specific folder structure. For now doing it this way, but this can be improved as previously discussed with @shailesh-vaidya, will raise with improvements in next pr.